README.md

```diff
@@ -107,14 +178,24 @@ Imagine that the squares of an ordinary checkerboard are numbered in two differe
 
 **a)**  Given a particular R and C, find the corresponding N.  That is, write an expression for variable N containing variables R and C.
 
+var r = 7;
+
+var c = 7;
+
+r * c = n + 7 * 2
+
 **b)**  Given N, find R.  Write an expression for R which contains N.
 
+n - 7 * 2 = r
+
```

Comment:
Again, we wouldn't want to define the variables by using numbers. For this one, we would use math.floor (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/floor). 

Basically, what this does is round the integer down to the nearest whole number. So, in this instance, we would divide the number by 8, and then use math.floor to round down to the nearest whole number, which would give us the correct column.  

Here is Dan's solution: r = Math.floor(n/8);